sparql/obsolete_replaced-by_term.rq

```diff
@@ -0,0 +1,21 @@
+#+ summary: Find replaced by term in obsolete term
+#+ description: Enter an obsolete term and return the term replaced by it 
```

Comment:
Change to:
`#+ description: Enter an obsolete term and return the term that replaces it`